Description: relax dependency on sequoia-wot
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

sequoia-sop appears to build and test fine with sequoia-wot 0.12.
When Debian updates to sequoia-wot 0.13, just drop this patch.

Forwarded: not-needed

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -59,7 +59,7 @@
 default-features = false
 
 [dependencies.sequoia-wot]
-version = "0.13"
+version = "0.12"
 default-features = false
 
 [dependencies.sop]
